transformers/senml/transformer.go

```diff
@@ -39,14 +39,20 @@ func (n transformer) Transform(msg broker.Message) (interface{}, error) {
 
 	msgs := make([]Message, len(normalized.Records))
 	for i, v := range normalized.Records {
+		// Use reception timestamp if SenML messsage Time is missing
+		time := v.Time
+		if time == 0 {
+			time = float64(msg.Created) / float64(1e9)
```

Comment:
Is this time in seconds (i.e. is this what Transformer uses)? If yes, a comment that explains this should be placed in front of this line.